controllers/storagecluster/cephcluster.go

```diff
@@ -1026,13 +1026,13 @@ func generateStretchClusterSpec(sc *ocsv1.StorageCluster) *rookCephv1.StretchClu
 		if zone == sc.Spec.NodeTopologies.ArbiterLocation {
 			continue
 		}
-		stretchClusterSpec.Zones = append(stretchClusterSpec.Zones, rookCephv1.StretchClusterZoneSpec{
+		stretchClusterSpec.Zones = append(stretchClusterSpec.Zones, rookCephv1.MonZoneSpec{
```

Comment:
@travisn is this change ok?

Reply:
This was necessary due to this https://github.com/rook/rook/pull/12384/commits/2231d73259707e082c24419bf68e4382444e3c6f#diff-46a2e53ea9b825b7c3b65113b63a133da71eb0cd2a2c0e33012c224d02a8eaa1

Reply:
Seems like a rename only, @travisn can confirm

Reply:
Correct, only the type name changed in a refactoring, but the json representation is the same and is not expected to affect upgrades